Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.5
```

Comment:
I think we discussed this, but there's no way (yet?) to move this to `swift/` yeah?

Reply:
No, the `Package.swift` has to be at the top-level.